### PR TITLE
documentation of filter.js

### LINKED ISSUE #119: Refractor the scripts
There are many script that are being reused a lot of times find them out and report them 

Reporting the issue will also be level 1 issue resolve

The level of the issue will depend on the difficulty of it

### PR DESCRIPTION
**Pull request by Preeti2095**
Partially Closes #119 

documented the filter.js file

### CODE REVIEW

**Comment by Preeti2095** [comment]
I have created a pull request for documentation of filter.js @AdityaTeltia. If there is any changes to be donw kindly let me know.

**Review by LordShenron — APPROVED** [review]
I suppose this one is for download page which is marked as unsupported (refer to #205 ) because we have a separate download page written completely in javascript ( refer to https://github.com/ProjectSakura/download ). 

Still I think we can merge this one so your hardwork don't go waste.

File: js/filter.js [hunk 1]
```diff
@@ -1,19 +1,54 @@
+/*
+This script is referenced in following files:-
+1.archive .html
```
Comment (ArunTeltia) [inline js/filter.js:3]: Delete this line

File: js/filter.js [hunk 2]
```diff
@@ -1,19 +1,54 @@
...
+2.blog.html
```
Comment (ArunTeltia) [inline js/filter.js:4]: This as well

File: js/filter.js [hunk 3]
```diff
@@ -1,19 +1,54 @@
...
 }
 
+/*
+below function is same as above with just change in parameters in few functions.
+
+ I) “input” variable is taking  html element with “id” value = ‘search123’
+ II) ‘ul’ variable is taking html element with “id” value = “search-ul1”. 
+ And rest is same.
+ */
+
```
Comment (ArunTeltia) [inline js/filter.js:51]: otherwise really great work @Preeti2095

**Review by ArunTeltia — CHANGES_REQUESTED** [review]

File: js/filter.js (hunk 3 above)
Comment (Preeti2095) [reply js/filter.js:51]: Delete this also?

**Review by Preeti2095 — COMMENTED** [review]
I have made the changes @ArunTeltia. If still there is changes to be made, kindly let me know.

**Review by yasharth291 — APPROVED** [review]
Looks good to me

**Review by ArunTeltia — APPROVED** [review]

**Comment by ArunTeltia** [comment]
Thank You Preeti

**Comment by Preeti2095** [comment]
Thank you @ArunTeltia for guiding me in this issue.

**Pull request merged**